test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/App.config

```diff
@@ -54,6 +54,18 @@
         <bindingRedirect oldVersion="0.0.0.0-2.7.41101.299" newVersion="2.7.41101.299" />
       </dependentAssembly>
 
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Data.Edm" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-5.8.4.0" newVersion="5.8.4.0" />
```

Comment:
I would expect these ODL-5.x dependencies already exist in the config and we just need to bump the versions to 5.8.4, instead of adding them?

Reply:
Removed. Thanks.